gmail/modules/listener/Module.md

```diff
@@ -231,3 +231,42 @@ service / on gmailEventListener {
 
 ```
 More samples are available at "https://github.com/ballerina-platform/module-ballerinax-googleapis.gmail/tree/master/samples/listener".
+
+> **NOTE:**
+At user function implementation if there was an error we throw it up & the http client will return status 500 error. 
```

Comment:
If the user's logic inside any remote method of the connector listener throws an error, connector internal logic will covert that error into a HTTP 500 error response and respond to the webhook (so that event may get redelivered), otherwise it will respond with HTTP 200 OK. Due to this architecture, if the user logic in listener remote operations includes heavy processing, the user may face HTTP timeout issues for webhook responses. In such cases, it is advised to process events asynchronously as shown below.